bin/bank/pycbc_brute_bank

```diff
@@ -187,11 +198,12 @@ class TriangleBank(object):
 
         msig = len(r)
 
-        #Apply tua0 threshold
+        #Apply tau0 threshold
         if args.tau0_threshold:
             hp.tau0 = pycbc.conversions.tau0_from_mass1_mass2(
                                             hp.params['mass1'],
-                                            hp.params['mass2'], 15.0)
+                                            hp.params['mass2'],
+                                            args.low_frequency_cutoff)
```

Comment:
I'm not opposed to allowing other options than 15 Hz, but I'm not sure it should always be the low frequency cutoff. Maybe add an option?